app/models/concerns/course/video/watch_statistics_concern.rb

```diff
@@ -33,6 +33,8 @@ def watch_frequency
 
   private
 
+  EVENT_TYPES = { start_times: ['play', 'seek_end'], end_times: ['pause', 'seek_start', 'end'] }.freeze
```

Comment:
The keys are shared between (1) hashes containing arrays of Course::Video::Events and (2) hashes containing arrays of events' video_time (integer). So if I change hash keys i have to change all of them, and then it wouldn't make sense for the second type of hashes. (See `filter_interval_events` definition.)

We can consider changing it to `:starts` and `:ends` so that it sounds relevant to both types of hashes.

Reply:
`start` and `end` then.